# https://bugs.launchpad.net/glance/+bug/1657459
# WebOb>=1.2.3 requirement for Glance will lead to 0 bytes backing image files
--- a/requirements.txt
+++ b/requirements.txt
@@ -10,7 +10,7 @@ eventlet!=0.18.3,>=0.18.2 # MIT
 PasteDeploy>=1.5.0 # MIT
 Routes!=2.0,!=2.1,!=2.3.0,>=1.12.3;python_version=='2.7' # MIT
 Routes!=2.0,!=2.3.0,>=1.12.3;python_version!='2.7' # MIT
-WebOb>=1.2.3 # MIT
+WebOb>=1.2.3,<1.7.0 # MIT
 sqlalchemy-migrate>=0.9.6 # Apache-2.0
 httplib2>=0.7.5 # MIT
 pycrypto>=2.6 # Public Domain

